platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -162,12 +168,17 @@ application to manage state changes and startup behavior. The `TSS Base Service`
 
 #### New Address Book Life-Cycle
 
-The TSS-Ledger-ID proposal modifies the address book and roster lifecycle presented in TSS-Roster as follows:
+Each day, on the first transaction after midnight, after the staking information has been updated by the app, it will
+generate a new roster and call a public API on `TSS Base Service`, which will then use this as the new candidate
+roster, and proceed with generation of TSS key material. This new roster, once keyed, is not used, unless there is
+a network upgrade.
 
-1. The `Roster` format does not change.
-2. The app will generate a candidate roster and generate the TSS key material for it before setting it as the next
-   candidate roster in the platform.
-3. The keyed candidate roster is always adopted on software upgrade, consistent with the TSS-Roster proposal.
+On the day the network is upgraded, the candidate roster will have started the keying process at midnight (UTC-0).
+The upgrade is usually performed during business hours, after 8:00 am (UTC-5). This is a 13-hour window for the
+key material to be computed. It is expected that this is sufficient time to compute the key material, but under
+certain conditions there is a possibility that the key material will not be computed in time. If the key material
+is not computed in time, the candidate roster will not be adopted and the network will continue to use the previous
+active roster.
```

Comment:
The wording at lines 179-181 slightly contradicts the more elaborate description of the changes to the roster life cycle, specifically the lines 716 and onwards below. I suggest to keep only a single copy of the life-cycle description in the document to ensure its consistency. Otherwise, please add some conditionals here to indicate when the lines 179-181 are actually true and when they aren't.

Reply:
I don't perceive the contradiction myself.  I am tweaking the wording here, hopefully fixing the perceived contradiction.   The different location of description is intended for different audiences.